### PR TITLE
Update teams-skype-interop.md - Missing pre-requisite of SRV record for Teams to SfC communications.

### PR DESCRIPTION
**Pull request by premgan**
Missing pre-requisite of SRV record for Teams to SfC communications.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-skype-interop.md [hunk 1]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
 
 ## Set whether Teams users can communicate with Skype users
 
-As an admin, you use the Microsoft Teams admin center or PowerShell to set external access settings to control whether Teams users in your organization can communicate with Skype users. By default, this capability is turned on for new tenants.
+As an admin, you use the Microsoft Teams admin center or PowerShell to set external access settings to control whether Teams users in your organization can communicate with Skype users. By default, this capability is turned on for new tenants. However there is a pre-requisite that the following DNS SRV record needs to be configured by the IT Admin if not already available for your domain. e.g. _sipfederationtls.contoso.com
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:71]:
```suggestion
As an admin, you use the Microsoft Teams admin center or PowerShell to set external access settings to control whether Teams users in your organization can communicate with Skype users. By default, this capability is turned on for new tenants. However, there is a pre-requisite that the following DNS SRV record needs to be configured by the IT Admin if not already available for your domain, e.g. _sipfederationtls.contoso.com.  
```

File: Teams/teams-skype-interop.md [hunk 2]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
 
 ## Set whether Teams users can communicate with Skype users
 
-As an admin, you use the Microsoft Teams admin center or PowerShell to set external access settings to control whether Teams users in your organization can communicate with Skype users. By default, this capability is turned on for new tenants.
+As an admin, you use the Microsoft Teams admin center or PowerShell to set external access settings to control whether Teams users in your organization can communicate with Skype users. By default, this capability is turned on for new tenants. However there is a pre-requisite that the following DNS SRV record needs to be configured by the IT Admin if not already available for your domain. e.g. _sipfederationtls.contoso.com  
+
+Service: sipfederationtls
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:73]:
```suggestion
Service: sipfederationtls <br/>
```

File: Teams/teams-skype-interop.md [hunk 3]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
...
+Protocol: TCP
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:74]:
```suggestion
Protocol: TCP <br/>
```

File: Teams/teams-skype-interop.md [hunk 4]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
...
+Priority: 100
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:75]:
```suggestion
Priority: 100 <br/>
```

File: Teams/teams-skype-interop.md [hunk 5]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
...
+Weight: 1
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:76]:
```suggestion
Weight: 1 <br/>
```

File: Teams/teams-skype-interop.md [hunk 6]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
...
+Port: 5061
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:77]:
```suggestion
Port: 5061 <br/>
```

File: Teams/teams-skype-interop.md [hunk 7]
```diff
@@ -68,7 +68,14 @@ Blocked Skype users, along with other people and public switched telephone netwo
...
+Target: sipfed.online.lync.com
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:78]:
```suggestion
**Target**: sipfed.online.lync.com
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**